scheduler/pkg/kafka/config/config.go

```diff
@@ -117,3 +117,30 @@ func (kc KafkaConfig) HasKafkaBootstrapServer() bool {
 	bs := kc.Consumer[KafkaBootstrapServers]
 	return bs != nil && bs != ""
 }
+
+// Based on config options defined for librdkafka:
+// https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md
+var secretConfigFields = map[string]struct{}{
+	"ssl.key.password":               struct{}{},
+	"ssl.key.pem":                    struct{}{},
+	"ssl_key":                        struct{}{},
+	"ssl.keystore.password":          struct{}{},
+	"sasl.username":                  struct{}{},
```

Comment:
Do we treat username also as secret?

Reply:
We don't have to - no strong feelings on this one, so I put it in deliberately to get feedback

Reply:
I guess we can consider it as such, it's important that we know it was was set to a value.